### ISSUE TITLE
Eliminate uses of deprecated CONSTANTS

### ISSUE THREAD

**Issue by leafpetersen** [dart-lang/fake_async#9]
This package still uses deprecated Dart 1 SCREAMING CAPS CONSTANTS. These will be removed shortly in Dart 2. This rewriting tool is available for the migration: This tool is available for the migration: https://github.com/dart-lang/dart2_fix .

**Comment by leafpetersen** [comment]
Oops, my bad.  It's actually the clock dependency which is broken, this package is ok.

**Comment by leafpetersen** [comment]
No, actually I was right the first time.  *sigh*.  Duration.ZERO is used in lib/fake_async.dart

**Comment by kevmoo** [comment]
@munificent just fixed this, I think
